app/views/govuk_publishing_components/components/docs/checkboxes.yml

```diff
@@ -114,7 +114,7 @@ examples:
         - label: "Blue"
           value: "blue"
   checkboxes_with_data_attributes:
-    description: Data attributes such as tracking can be applied if required. Note that for a checkbox this will not fire tracking events automatically, so if you want your tracking to work you'll need to write some tracking Javascript in the app around the component to detect it.
+    description: Data attributes such as tracking can be applied if required. This will fire tracking events.
```

Comment:
We should probably make it clear that this will fire an identical event for checking and unchecking, and then point people to the example below if they need to be able to distinguish between the 2.